pages/content/amp-dev/documentation/guides-and-tutorials/develop/animations/triggering_css_animations.md

```diff
@@ -9,7 +9,7 @@ formats:
 
 Triggering CSS animations [relies on adding and removing classes](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Animations/Using_CSS_animations), done via JavaScript. You can achieve the same behavior on AMP pages by using the [`toggleClass`](/content/amp-dev/documentation/guides-and-tutorials/learn/amp-actions-and-events.md#-all-elements)) action. With `toggleClass`, AMP pages can add, remove, and toggle CSS classes like non-AMP pages.
 
-## Defining CSS and Keyframes
+## Defining CSS and keyframes
```

Comment:
I saw there is a closing bracket to much at the end of the link `amp-actions-and-events.md#-all-elements` above.

Reply:
nice catch - thanks!